third_party/move/move-compiler/src/naming/translate.rs

```diff
@@ -353,6 +353,9 @@ pub fn program(
     prog: E::Program,
 ) -> N::Program {
     let mut context = Context::new(compilation_env, pre_compiled_lib, &prog);
+    if context.env.flags().get_block_v1_compiler() {
```

Comment:
I really like this idea - simple, but would have saved me some time wondering "are we sure the compiler v2 ran" in a few circumstances.